packages/app/src/vis-packs/core/visualizations.ts

```diff
@@ -72,7 +75,7 @@ export const CORE_VIS = {
     Icon: FiCpu,
     Container: RawVisContainer,
     ConfigProvider: RawConfigProvider,
-    supportsDataset: hasNonNullShape,
+    supportsDataset: hasScalarShape,
```

Comment:
This is just to avoid complexifying the fallback visualization code in the `Visualizer` for now. As mentioned, the plan is to merge `RawVis` into `ScalarVis`.